.changes-pending.md

```diff
@@ -9,8 +9,13 @@
 * fix: reword comment (900300 config) (Christian Folini) [#3417]
 * feat: improve detection by adding missing javascript `prompt` and `confirm` methods (941390 PL1) (Jitendra Patro) [#3395]
 * fix: handle false positive by fixing whitespace matching after PHP command (933160 PL1) (Max Leske) [#3432]
-* docs: Added link to run tests (Ervin Hegedus) [#3438]
+* docs: add link to run tests (Ervin Hegedus) [#3438]
 * fix: improve rule by matching non-word-boundary of commands with options (932237 PL3) (Max Leske) [#3425]
 * fix: replace backend docker container for tests to fix JSON Unicode reflection (Max Leske) [#3464]
-* fix: Unwanted match of character set substrings in multipart/form-data requests (922100 PL1) (Jozef Sudolský) [#3470]
-* fix: FPs with word "settings" (932236 PL2, 932237 PL3, 932239 PL2) (@EsadCetiner) [#3394]
+* feat: add .vscode to restricted-files.data (930130 PL1) (Frederik Himpe) [#3471]
+* feat: remove redundant t:lowercase for a little performance (922110 PL1) (Jozef Sudolský) [#3469]
+* fix: update comments (922110 PL1, 942440 PL2) (Jozef Sudolský) [#3468]
+* feat: add unix commands pyversions and py3versions (932235 PL1, 932260 PL1, 932236 PL2, 932237 PL3, 932239 PL2) (Jitendra Patro) [#3465]
+* feat: add new test method: check for tags on rules against allowlist (Ervin Hegedus) [#3437]
+* fix: refine unwanted match of character set substrings in multipart/form-data requests (922100 PL1) (Jozef Sudolský) [#3470]
```

Comment:
```suggestion
* fix: unwanted match of character set substrings in multipart/form-data requests (922100 PL1) (Jozef Sudolský) [#3470]
```

Reply:
I try to have a "verb" at the beginning of every entry. This was the best one I could find.

You insist we should avoid it here?

Reply:
There you go :)

Reply:
Brillant